client/app/telecom/pack/xdsl/access/pack-xdsl-access.html

```diff
@@ -288,8 +290,30 @@ <h5 data-translate="xdsl_details_NRA_title"></h5>
                                     </dl>
                                 </div>
                             </li>
-
-
+                            <!--NRO-->
+                            <li class="oui-tile__item" data-ng-if="access.xdsl.isFiber">
+                                <div class="oui-tile__definition" data-ng-if="access.xdsl.isFiber">
+                                    <h5 data-translate="xdsl_details_NRO_title"></h5>
```

Comment:
`h4`, `h3`, `h2` and `h1` above ? If not, use the correct level and change the displaying according to what you want